src/k8s/pkg/k8sd/types/cluster_config_validate.go

```diff
@@ -20,6 +21,24 @@ func validateCIDRs(cidrString string) error {
 	return nil
 }
 
+// isValidAddress checks if a given address is valid. Allows with and without schema.
+func isValidAddress(address string) bool {
+	if u, err := url.Parse(address); err == nil && u.Host != "" {
+		return true
+	}
+
+	host, port, err := net.SplitHostPort(address)
+	if err == nil {
+		if host == "localhost" || net.ParseIP(host) != nil {
```

Comment:
so?  no fqdn for etcd servers are allowed?  

example)
`172.22.1.1.ec2.internal`  would be disallowed?

Reply:
loosened the check to allow fqdn and extended unittests